packages/twenty-front/src/pages/settings/security/SettingsSecurity.tsx

```diff
@@ -23,7 +23,18 @@ export const SettingsSecurity = () => {
     >
       <SettingsPageContainer>
         <Section>
-          <H2Title title="SSO" description="Configure an SSO connection" />
+          <H2Title
+            title="SSO"
+            description="Configure an SSO connection"
+            addornment={
+              <Tag
```

Comment:
I feel like in future iteration it will be worth abstraction this into something re-usable but we can do it later!